docs/2.8.4/docs/canton/usermanual/apis.rst

```diff
@@ -295,6 +280,41 @@ certificates of the Java runtime will be used. An example would be:
    :end-before: architecture-handbook-entry-end: TlsConnect
    :dedent:
 
+Server Authentication
+^^^^^^^^^^^^^^^^^^^^^
+
+Canton has two ways to perform server authentication to protect from man in the middle attacks. Firstly, if TLS is used on the public API as described above, TLS also takes care of server authentication as that is one of the core functions of TLS.
+
+Secondly, like all nodes, the sync domain has an identity which corresponds to the fingerprint of its namespace root key. It reports its identity to connecting participant nodes and signs all its messages with keys authorized by that namespace root key on the topology ledger. Assuming no key compromises, this gives participants a guarantee that the reported identity is authentic. Thus server authentication can also be performed by the sync domain operator passing their sync domain identity to the participant node operator, and checking that the identity matches that reported by the sync domain to the participant node. The domain identity of the sole connected sync domain can be read out using console commands like:
+
+.. code-block:: none
+  
+    participant1.domains.list_connected.last.domainId.filterString
+
+Client Authentication
+^^^^^^^^^^^^^^^^^^^^^
+
+Unlike the admin api, the public API uses Canton's cryptography for client authentication rather than mTLS. To connect to the public APIs, clients need to connect in two steps:
+
+1. Call the `SequencerConnectService` to align on Canton Protocol versions.
+2. Call the `SequencerAuthenticationService` to authenticate and get an access token for the other services.
+3. Connect to the main `SequencerService` using the access token from 1.
+
+During stage 2, the client claims an identity, which is the fingerprint of a namespace root key. If that identity is allowed to connect at all (see :ref:`permissioned-domains`), then the domain responds with a challenge consisting of a nonce and all known key fingerprints of keys that are allowed to authenticate for that identity as per the topology ledger. During the first connection, that would be the namespace root key alone.
```

Comment:
last sentence: it's actually the (transaction) signing key of the node, not the namespace root key, that is used to sign the challenge.